docs/logo/fizzy_logo_turquoise.svg

```diff
@@ -2,25 +2,30 @@
 <!-- Generator: Adobe Illustrator 24.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
This comment may be dropped as not accurate any more.

Reply:
Removed.